docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+services:
+  postgres:
+    image: pgvector/pgvector:pg17
+    environment:
+      - POSTGRES_USER=ecl
+      - POSTGRES_PASSWORD=ecl
+      - POSTGRES_DB=enthusiast
+  redis:
+    image: redis:latest
+  frontend:
+    build:
+      context: frontend/
+      dockerfile: Dockerfile
+    volumes:
+      - ./frontend/src:/app/src
+    environment:
+      - PORT=10001
+      - VITE_API_BASE=http://localhost:10000
+      - VITE_WS_BASE=ws://localhost:10000
+    ports:
+      - "10001:10001"
+  api:
+    build:
+      context: server/
+      dockerfile: Dockerfile
+    volumes:
+      - ./server:/app
```

Comment:
Do you think we can remove this volume? As it probably won't be needed in "production" like docker-compose